scylladb/sphinx-scylladb-theme
# fix: legacy panel box (dark theme)

Closes #1236

## How to test

1. Clone the PR & build the docs.
2. Open **Examples** > **Panel box**.
3. Check "Compatibility with previous versions" example renders without errors in dark mode.
